tests/lib/tools/vmcli/vmcli/main.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2024 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.import pkg_resources
+import argparse
+import asyncio
+import asyncio.exceptions
+import logging
+import pathlib
+from typing import Any
+
+import telnetlib3
+import telnetlib3.client
+
+log = logging.getLogger("main")
+
+
+def parse_arguments() -> Any:
```

Comment:
You can replace `Any` with `argparse.Namespace`, I believe. It has some smarts to understand that better AFAIR.